app/controllers/account/admin/issues_controller.rb

```diff
@@ -13,7 +14,8 @@ def edit; end
 
       def update
         if @issue.update_attributes(issues_params)
-          flash[:success] = 'Issue updated'
+          flash[:success] = 'Звернення змінено!'
+          facebook_posting(@issue) if @issue.status == 'open'
```

Comment:
so every time when you will update issue (in state == open) it will be posted to FB?